### ISSUE TITLE
[URGENT] Filepicker broken

### ISSUE THREAD

**Issue by adelq** [pennlabs/eventsatpenn-meteor#61]
All images not working IN PRODUCTION

@jasontsh @dlakata

**Comment by yefim** [comment]
I think it was using my S3 bucket to upload to. Can you migrate it to another bucket?

**Comment by adelq** [comment]
Yes, we should, didn't know it was going to your bucket @dlakata
